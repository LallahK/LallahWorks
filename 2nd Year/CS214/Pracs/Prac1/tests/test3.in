23840441
23465338
22171517
23699164
21031371
24474091
23444870
22996407
21316975
21800364
24713185
27031849
28421070
27889337
23653945
27668206
25777158
29000391
20901156
23553166
21946572
21965684
27867575
20231507
28016209
25149780
28917575
28463654
21572958
21476734
22334837
23719906
28621054
27260791
25501615
21625072
24101289
29998994
24445839
20856900
24038563
25269602
21028179
28483606
24459161
24809757
28555371
21841630
25693544
24836175
