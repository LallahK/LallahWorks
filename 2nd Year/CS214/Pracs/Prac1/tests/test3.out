683
925
953
1291
594
1075
752
1200
457
285
463
637
564
1208
820
964
1086
384
276
832
756
959
1028
524
451
828
539
974
1002
1015
608
442
550
970
286
538
342
1382
741
777
688
950
570
1126
896
402
909
664
1235
684
